src/screens/EventVolunteers/VolunteerGroups/modal/VolunteerGroupModal.module.css

```
@@ -0,0 +1,58 @@
+  border-radius: var(--radius-sm);
+  background-color: var(--color-blue-700);
+  width: 100%;
+  font-size: var(--font-size-md);
+  color: var(--color-white);
+  outline: none;
+  font-weight: var(--font-weight-semibold);
+  cursor: pointer;
+  transition:
+    transform 0.2s,
+    box-shadow 0.2s;
+  width: 100%;
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

width is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)